### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by flagarde**
Hi, this PR exposes the lightweightness of the library and list most of the CI we are doing. IMHO some basic hello world could give a taste of this library.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,18 +9,66 @@
...
+The minimal Hello World is as simple as :
+
+```cpp
+#include "cpp-terminal/io.hpp"
+
+#include <iostream>
+
+int main()
+{
+  std::cout << "\033[31mHello world !\033[0m" << std::endl;
```
Comment (MCWertGaming) [inline README.md:26]: what happens on an exception? Does the terminal revert itself back properly?

**Review by MCWertGaming — CHANGES_REQUESTED** [review]
I added a smaller question, but I'd like to improve the grammar of the first sentence in the next days, i hope you don't mind ^^

File: README.md (hunk 1 above)
Comment (flagarde) [reply README.md:26]: Please. I skipped grammar classes indeed 😂. French and english is really not a good mix

Comment (flagarde) [reply README.md:26]: @MCWertGaming If I remember well the C++ rules : 
```
If an exception is thrown and not caught, including exceptions that escape the initial function of [std::thread](https://en.cppreference.com/w/cpp/thread/thread), the main function, and the constructor or destructor of any static or thread-local objects, then [std::terminate](https://en.cppreference.com/w/cpp/error/terminate) is called. It is implementation-defined whether any stack unwinding takes place for uncaught exceptions.
``
So it's depends :(

Comment (MCWertGaming) [reply README.md:26]: > Please. I skipped grammar classes indeed joy. French and english is really not a good mix

I can tell, was the same to me with french class :p


> @MCWertGaming If I remember well the C++ rules :
> 
>     * If an exception is thrown and not caught, including exceptions that escape the initial function of [std::thread](https://en.cppreference.com/w/cpp/thread/thread), the main function, and the constructor or destructor of any static or thread-local objects, then [std::terminate](https://en.cppreference.com/w/cpp/error/terminate) is called. It is implementation-defined whether any stack unwinding takes place for uncaught exceptions.
> 
> 
> So it's depends :(

Alrighty ^^

**Comment by MCWertGaming** [comment]
@flagarde I can't push directly into this branch somehow, so I created a PR: https://github.com/flagarde/cpp-terminal/pull/1

**Comment by MCWertGaming** [comment]
The PR Will also fix the CI

**Comment by flagarde** [comment]
Merged

**Comment by flagarde** [comment]
@MCWertGaming Maybe you can adapt the description of the `About` section on https://github.com/jupyter-xeus/cpp-terminal/ which still says : 
```
About

Header-only C++ library for writing multiplatform terminal applications 
```

**Comment by MCWertGaming** [comment]
> @MCWertGaming Maybe you can adapt the description of the `About` section on https://github.com/jupyter-xeus/cpp-terminal/ which still says :
> 
> ```
> About
> 
> Header-only C++ library for writing multiplatform terminal applications 
> ```

Yeah, that bothers me, but I haven't permissions to change that either. @certik is it possible that you give me a higher permission role on the repository or to change the repository descriptions? Would also be nice to have a few repository tags.

**Review by MCWertGaming — APPROVED** [review]
This should be fine now ^^ Thank you for the work @flagarde.

**Comment by certik** [comment]
@MCWertGaming I just gave you Maintainer permission, let me know if you can now change the repository description.

**Pull request merged**